server/src/assembly/resources/conf/iotdb-engine.properties

```diff
@@ -451,11 +451,6 @@ timestamp_precision=ms
 # If you are feeling the rebooting is too slow, set this to false, false by default
 # continue_merge_after_reboot=false
 
-# When set to true, all unseq merges becomes full merge (the whole SeqFiles are re-written despite how
-# much they are overflowed). This may increase merge overhead depending on how much the SeqFiles
-# are overflowed.
-# force_full_merge=true
-
```

Comment:
remove the argument in `IoTDBConfig` and `IoTDBDescriptor`

Reply:
resolved